spec/api-app-spec.js

```diff
@@ -122,6 +122,12 @@ describe('app module', () => {
     })
   })
 
+  describe('app.getLocaleCountryCode()', () => {
+    it('should be empty or have length of two', () => {
+      expect(app.getLocaleCountryCode().length).to.be.oneOf([0, 2])
```

Comment:
So if the function will suddenly start returning an empty string all the time, the test won't catch it?
Is there a way to predict its return value? Does it depend on a platform?

Reply:
@zarubond 
Can you please the expected length to `0` only for the linux CI? Like this:

```js
const expectedLength = 2
if (isCI && process.platform === 'linux') {
  // Linux CI machines have no locale.
  expectedLength = 0
}
expect(app.getLocaleCountryCode()).to.be.a('string')
    .and.have.lengthOf(expectedLength)
```

Reply:
@zarubond 
Can you please set the expected length to `0` only for the linux CI? Like this:
```js
let expectedLength = 2
if (isCi && process.platform === 'linux') {
  // Linux CI machines have no locale.
  expectedLength = 0
}
expect(app.getLocaleCountryCode()).to.be.a('string')
    .and.have.lengthOf(expectedLength)
```

Reply:
Added the exception for Linux.